### PR TITLE
Issue 2 further db improvements

### PR DESCRIPTION
**Pull request by artofimagination**
Need to merge #4 before checking this

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -154,7 +155,11 @@ func main() {
 	http.HandleFunc("/get-settings", getSettings)
 	http.HandleFunc("/delete-settings", deleteSettings)
 
-	if err := mysqldb.BootstrapSystem(); err != nil {
+	mysqldb.DBConnection = "root:123secure@tcp(user-db:3306)/user_database?parseTime=true"
```
Comment (pongi1987) [inline main.go:158]: Pwd in the code  🤦‍♂️

File: models/product.go [hunk 2]
```diff
@@ -0,0 +1,53 @@
+package models
+
+import (
+	"github.com/google/uuid"
+)
+
+type Product struct {
+	ID      uuid.UUID `validation:"required"`
+	Name    string    `validation:"required"`
+	Public  bool      `validation:"required"`
```
Comment (pongi1987) [inline models/product.go:10]: Simple bool couldn't be omited or required.
If you really would like to make bool values required, you should use pointers here:
```suggestion
	Public  *bool      `validation:"required"`
```

File: mysqldb/products_test.go [hunk 3]
```diff
@@ -0,0 +1,403 @@
+package mysqldb
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/artofimagination/mysql-user-db-go-interface/models"
```
Comment (pongi1987) [inline mysqldb/products_test.go:7]: Are you using `goimports`?

**Review by pongi1987 — CHANGES_REQUESTED** [review]

**Comment by artofimagination** [comment]
Not needed since reworks in #7

**Pull request closed without merging**